#Ibarra-Laclette

## Intro

C-value paradox, junk DNA, ENCODE

Polyploidy, function, genome size.

Vocab:

* fractionation <strong style="font-size: 150%; color: red;">?</strong>
* homolog -- why not use ortholog <strong style="font-size: 150%; color: red;">?</strong>

Sequencing and Genome build

* why different technologies and sizes <strong style="font-size: 150%; color: red;">?</strong>
* assembly
* transcriptome: why different tissues <strong style="font-size: 150%; color: red;">?</strong>
* why transcriptome useful <strong style="font-size: 150%; color: red;">?</strong>
* How estimate Ne <strong style="font-size: 150%; color: red;">?</strong>
* Why did they want to estimate Ne <strong style="font-size: 150%; color: red;">?</strong>

## Results

* Good assembly
* Same or fewer genes than other plants <strong style="font-size: 150%; color: red;">?</strong>
* Does this support the c-value paradox <strong style="font-size: 150%; color: red;">?</strong>

Show synteny in figure 1. and figure 2.

* 8x with respect to ancestor which is itself what ploid <strong style="font-size: 150%; color: red;">?</strong>
* what do you expect from 8x vs. 2x in terms of genome size <strong style="font-size: 150%; color: red;">?</strong>

What are two ways it gets a smaller genome <strong style="font-size: 150%; color: red;">?</strong>

* fractionation
* fewer transposable elements (Has not lost ability to silence TEs)
* even introns affected!

So small genes overlap! Tail-to-tail (draw)

Use transient expression analysis (will come back to) to prove that remaining intergenic is largely doing something (regulating expression)

Also show microRNAs which regulate stuff.

We covered fractionation, but how lose TEs<strong style="font-size: 150%; color: red;">?</strong>

Draw recombination deletetion as in class.

So it got smaller. Could this be random mutation <strong style="font-size: 150%; color: red;">?</strong>

Unlikely because they estimated Ne was large(ish) so s>>1/(2Ne)

### Why care about orthology?

Want to infer function in most of these genes.  If they can show clear orthology, similar function is a good bet!

* e.g. Casparian strip -- mostly found in roots, but this has no roots -- likely sub or neo function!

* e.g. identification of MADS box genes which are important developmental regulators
