# Kudla et al 2009

Motivation: understand codon bias.

Hyp. 1: using common codons will allow higher expression

* Sub: CAI in only part of gene?

Hyp. 2: codon usage will affect RNA folding and expression levels

* Sub: Only in part of gene?

## Sequence design and vector

Fig. 1: create lots of pieces of gene with degenerate nucleotides. These are nucleotides that can pair with multiple other nucleotides

PCR amplify these: what will happen <strong style="font-size: 150%; color: red;">?</strong> Will get randomized nucleotides at 3rd position

Put these into vector, with T7 promoter which is very specific for T7 polymerase (thus high expression). Insert into bacteria with T7 polymerase -> high expression.

F1 is origin of replication

MCS is a site engineered with lots of restriction enzyme (what is<strong style="font-size: 150%; color: red;">?</strong> )sites so you can insert sequence easily. 

What's the amp <strong style="font-size: 150%; color: red;">?</strong> To know your transformation worked.

Why the lacI <strong style="font-size: 150%; color: red;">?</strong> So you can induce transcription by putting bacteria on medium with low lactose and high galactose!

How to calculate CAI: show math. baseline is comparison of highly expressed gene set (a bit circular!)

Fig1: their genes representative of range seen in E. coli.

High CAI means <strong style="font-size: 150%; color: red;">?</strong> codon w/ lots of tRNA. 

## Experimental setup

What's a biological replicate <strong style="font-size: 150%; color: red;">?</strong> vs. technical. (weighing tiger vs. great white sharks example) So measuring same insertion event multiple times/ways -> technical. Repeating insertion event -> biological replicate. 

Why test alternate bacterial promoter <strong style="font-size: 150%; color: red;">?</strong> Rule out some T7 specific promoter effect. Result <strong style="font-size: 150%; color: red;">?</strong> about the same

What does measuring flourescence get them <strong style="font-size: 150%; color: red;">?</strong> protein abundance. Does this get mRNA levels <strong style="font-size: 150%; color: red;">?</strong> No.

Test multiple ways of measuring flourescense -> all same. Show figure S4.

Also test western blot <strong style="font-size: 150%; color: red;">?</strong> separate proteins on gel and use antibodies to identify

## Energy stuff

Can measure "free energy" of molecule. Basically how much more stuff could it do. If it's 0, that means it's pretty stable and can't do anything more. Show figure 2. RNA folding.

## Results

CAI nor freq. optimal codons correlate w/ fluorescence. (Why use multiple measures <strong style="font-size: 150%; color: red;">?</strong> )

Rare codons, rare pairs, also no correlation. Nor did CAI of first amino acids. Figure 2A.

**Conclusion**: nada

Ask if position of codon makes a difference. For basic codon usage model, should it <strong style="font-size: 150%; color: red;">?</strong> Figure S6

Suggests mRNA structure.

Measure free energy. And <strong style="font-size: 150%; color: red;">?</strong> total was not correlated but folding of first 1/3 was. Fig. 2B and 2C.

Sliding windo analysis showed first part explained most of variation under both promoters.

**Conclusion**: tightly folded obstructs initiation of translation. Where else do we see hairpin loops <strong style="font-size: 150%; color: red;">?</strong>  Transcription termination and initiation in tryp operon.

If blocking initiation, why didn't Shine-Delgarno sequence show up <strong style="font-size: 150%; color: red;">?</strong> (what is S-D <strong style="font-size: 150%; color: red;">?</strong> region just upstream of translation start site for ribosome binding) : their sequences all had same S-D!

mRNA levels as measure by Northern blot, but variaiton 3X less. Why <strong style="font-size: 150%; color: red;">?</strong> codons affect transcription less than translation. But still correlated w/ flourescence (more mRNA -> more protein) and affected by mRNA folding. What's interpretation <strong style="font-size: 150%; color: red;">?</strong> mRNA folding affects ribosome binding. less ribosome = less translation, also = more exposure to digestion, so mRNA lasts less time so observe fewer transcripts.

Prediction: adding weak 5' mRNA folding sequence should increase expression even if CAI not changed. Fig. 3

## However

CAI = faster growth! mRNA folding noT! Why <strong style="font-size: 150%; color: red;">?</strong>   Slower translation uses up ribosomes, so overall cell process has to wait longer. 

**Conclusions**: So instead of high CAI -> high expression (because fast); high expression -> high CAI (because you don't want to use up all the ribosomes)


