# BIS101 F2013 Lecture 7: Macroevolution and Phylogenetics

## Notes

Instead do echolocation paper!

## Reading

All of Ch. 17. Check syllabus for updates. HW and reading updates as we go along
 

## Phylogenetics

Study of relationships among genes and species. Important for understanding not just how things are related, but using that information to learn about function, mechanistic basis of function, both of genes and phenotypes.

Go through C4 tree.

Then bat/dolphin. Draw out inference if dolphin/bat closest. One or multiple <strong style="font-size: 150%; color: red;">?</strong>

Then add cow and ask gain vs. loss <strong style="font-size: 150%; color: red;">?</strong>

Then look at real phylogeny. Gain multiple times. So similar or differen genetic basis <strong style="font-size: 150%; color: red;">?</strong>

Predict different, but in this case similar. What does that tell you? Limited number of genes that can be changed to gain echolocation. Actually same parts of some of these genes changed in same way!

### Inferring function:

How do know how much of the genome is functional?

Well, let's look at a gene, makes a transcript, and a protein.  Functional <strong style="font-size: 150%; color: red;">?</strong>

Human exome (<strong style="font-size: 150%; color: red;">?</strong>) is 100Mb (how much <strong style="font-size: 150%; color: red;">?</strong> ).  But human genome is <strong style="font-size: 150%; color: red;">?</strong> (3Gb)

What about the rest? 

Draw human, chimp, orangutan, cow.  Draw a sequence present in only human and one present in all of them.  Which more likely to be functional <strong style="font-size: 150%; color: red;">?</strong> 

## How to build?

Usually use gene sequences to determine relatedness.  For example:

Raccoon      

    ATGGGA

Bear        

    ATGGGA

Cat         

    AACGGT

Mongoose    

    AACGGA

Platypus    

    GTCTTA

**Parsimony** (<strong style="font-size: 150%; color: red;">?</strong>) method.  Method that requires the fewest changes -- the simplest.

Draw one wrong tree.  Redraw better tree.

Really people use whole genomes and very complex statistical methods. We will not go into these details. 

Draw tree, label nukes. Define **outgroup** allowing you to put an ancestral root on the tree.

Does this mean platypus is "basal" or "ancestral" to all mammals <strong style="font-size: 150%; color: red;">?</strong> No. Explain same distance

## Gene tree vs. species tree

This has assumed all Cats have one sequence.  This works for distantly related lineages where all differences are fixed. 

But populations are polymorphic for different gene sequences. And multiple alleles segregating.

Draw species tree of humans, chimps, gorilas.

<img src="/Users/jri/Documents/courses/bis101/fall2014/images/venema_20_7.jpg" style="width: 400px;"/>

We actually predict this to be common. 
In fact 25% genes are more related to gorilla than chimp.  And 1% closer to orangutan than chimp!


## Gene duplication

**Homolog** closely related gene.

Genes can be copied and duplicated over time, and gene families can grow and shrink. 

This leads to two kinds of homologs.

**ortholog** vs **paralog**

separated by speciation vs. separated by duplication

<img src="/Users/jri/Documents/courses/bis101/fall2014/images/ortholog.jpg" style="width: 400px;"/> 

But draw star earlier.  Key to difference is idnetifying where on tree duplications have occurred.  Then just ask how genes are seaparated by tracing back.

Remove human and ask ortholog/paralog.

Draw genes at tips and at duplication event

define **synteny**

draw how synteny can inform which gene is ortholog.

fates of a new gene duplicate: 
Subfunctionalization vs. neofunctionalization vs. pseudofunctionalization

what would happend to a pseudogene over time?   

## Gene evolution

What will happen to new deleterious mutations <strong style="font-size: 150%; color: red;">?</strong> 

**Purifying selection** removes new deleterious mutations. Important genes with functions will be under stronger <strong style="font-size: 150%; color: red;">?</strong> or weaker purifying selection?

How to identify.

Define **nonsynonymous** and **synonymous** changes<strong style="font-size: 150%; color: red;">?</strong>

Which is more likely to be functional <strong style="font-size: 150%; color: red;">?</strong>

Which is more likely to be deleterious <strong style="font-size: 150%; color: red;">?</strong>

If both have no effect on function, expect equal proportions of such changes.  

Can compare gene sequences along a phylogeny and count nonsyn and syn changes per bp.  if dN/dS << 1, constraint!  if dN/dS >> 1, rapid adaptation!


