# BIS101 F2013 Lecture 11: RNA
## Reading

## RNA

ribose sugar instead of deoxyribose (extra OH group)
 
single-stranded. consequences <strong style="font-size: 150%; color: red;">?</strong>	 so molecule can bind to itself, more 3D structure
        
RNA does not have thymine, but has uracil instead. uracil can bind to both adenine and guanine (weakly) which allows some of the tertiary structure (and wobble we will come to later)

can catalyze biological reactions â€” ribozyme

* this fact led to inference of RNA world

## Kinds of RNA

transfer RNA (tRNA) <strong style="font-size: 150%; color: red;">?</strong>

* bring amino acid to mRNA in a ribosome for appropriate translation

messenger RNA (mRNA) 

* only kind that is translated -> protein

ribosomal RNA (rRNA) <strong style="font-size: 150%; color: red;">?</strong>

* major components of ribosomes, which are the molecular machines that assemble amino acids to form proteins from mRNA

small nuclear RNA (snRNA)

* important for RNA processing inside the nucleus
* some form part of spliceosome (what's that do?)

micro RNA (miRNA) <strong style="font-size: 150%; color: red;">?</strong>

* regulate protein production 

long noncoding RNA ncRNA or lncRNA

* function not well known. book says most of genome codes for RNA, this is not really true at any meaningful level. 

tRNA, snRNA, rRNA **constitutively** <strong style="font-size: 150%; color: red;">?</strong> transcribed. others transcribed as needed when/where needed
        
## Transcription

DNA -> RNA called <strong style="font-size: 150%; color: red;">?</strong>

Spatial separation <strong style="font-size: 150%; color: red;">?</strong> Transcription happens in nucleus, translation in cytoplasm. True in prokaryotes <strong style="font-size: 150%; color: red;">?</strong> no.

RNA product of transcription = transcript

Transcription occurs 5'->3'

Occurs from **template strand**. DRAW

<img src="/Users/jri/Documents/courses/bis101/griffiths/ch08/ch08/figure_08_03.jpg" style="width: 300px;"/>

That means transcript is complementary to template strand and matches the **non-template** or **coding strand**. Which is which differs for each gene.

Genes usually drawn on coding strand. Each gene coded for only on one strand, always same. Practice on Figure 8-6 to get used to this.

RNA polymerase transcribes DNA into RNA. 3 main parts to transcription: initiation, elongation, termination.

## Initiation

RNA polymerase binds to promoter region with help of proteins.

* Why important to have other proteins that bind to DNA <strong style="font-size: 150%; color: red;">?</strong> 

In prokaryotes, different protein subunits of the complete package allow RNA polymerase to bind to different promoter sequences in the DNA. Otherwise could only bind to one promoter or would need multiple polymerases.

In eukaryotes, not just polymerase binding to promoter. Whole suite of proteins necessary to bind. Called **General Transcription Factors**. These bind and attract polymerase. Polymerase moves on and these can stick around to attract more polymerases so the gene can be continuously transcribed -- multiple copies being transcribed at same time (Figure 8-5 is cool).

initiation site near 3' end of promoter. First sequence transcribed but not translated (translation starts at ATG codon) is called 5' untranslated region. This is first EXON!

**DRAW** basic gene structure on board, with exon/intron. Add 5' UTR show first exon.

**Upstream** = 5' and negative numbers and **downstream** = 3' or positive numbers.

Promoter sequence in eukaryotes often (~25%) has TATA ~30 nukes upstream of transcription start site (TSS), called **TATA BOX**, important to help RNA polymerase position correctly



## Elongation

DRAW line of transcript along whole gene. Transcription occurs 5'-3' of the transcript, so polymerase moves 3'->5' on the DNA.

RNA Polymerase makes transcription bubble where unwinds DNA, adds ribonucleotides, then those are separated and string off as they exit the polymerase.

In euks, multiple polymerases for different RNA. 

## Termination

Transcription ends at region providing termination signal, beyond last bit of translated sequence (which ends in what<strong style="font-size: 150%; color: red;">?</strong> stop codon). This makes for 3' UTR and last exon!

In prokaryotes, termination occurs one of two ways. sequences at end of RNA make a hairpin loop (binds to itself, draw) followed by polyadenine tract that is less stable. RNA pol hits this unstable and backs up, runs into loop, transcript falls off.

Or rho factor protein binds to specific sequence and does similar effect.

In eukaryotes polyadenylation signal AAUAAA or AUUAAA (polyadnenylation signal) near end which is recognized and cut.

## Processing

In eukaryotes, processing of RNA occurs while being transcribed. **primary transcript** or **pre-mRNA** is processed into mature mRNA. Processing orchestrated by carboxyl tail domain (CTD) of RNA pol.

adds 7-methylguanosine to 5' end. called **cap**. protects from degradation and needed for translation.

processing includes adding a poly-A (~150 A's) tail to end of 3'

### splicing. 

* genes can be 100's of kb long but only a few kb in mature mRNA!
* ~3/4 of human genes alternatively spliced, lower in plants
* splicing means 20K genes (how many in humans <strong style="font-size: 150%; color: red;">?</strong> ) can make ~100K proteins!

**splicesome** made up of proteins and snRNAs that bind to specific intron sequences.

splice site acceptor and donor GU AG in pre-mRNA, GT AG in coding (yes) or template (no) strand <strong style="font-size: 150%; color: red;">?</strong> 

recognizes these sites, loops out intron, and cuts. 

on 3' end adds string of 150+ adenine nucleotides **polyA tail**

