# BIS101 F2013 Lecture 9: DNA

## Exam stuff

## Notes/Questions

## Paper

## Q1

## Q2

## Q3


#DNA 

DNA is the heritable material. Book goes into the clever experiments that showed this.

## Structure

DNA is pretty simple. Keep in mind it's a physical molecule, has propertie, takes up space, and these properties have a lot of implications for mutation, for how proteins interact with DNA, how it can be copied, etc..

* phosphate
* doxyribose sugar
* nitrogenous bases <strong style="font-size: 150%; color: red;">?</strong> -- adenine guanine, cytosine, thymine, uracil, and hypoxanthine
    - uracil and hypoxanthine mostly in RNA and tRNA

Purines: adenine and guanine (and caffeine, theobromine and uric acid!)
Pyrimidines: thymine and cytosine


**Nucleotide** is the combination of a base (ATGC) a doexyribose sugar, and a phosphate group (draw 1). You remove the phosphate you have a **nucleoside**

<img src="/Users/jri/Documents/courses/bis101/griffiths/ch07/figure_07_05.jpg" style="width: 500px;"/>

DNA is double stranded with two strands of nuleotides wound together in double helix. It's a right-handed structure (show fist) although left-handed DNA exists (z-DNA) it is rare. Laugh at people who draw the helix wrong.

**Major groove**  and **minor groove**

Backbone is made of phophate group of one nucleotide linked to ribose of the next via a **phosphodiester bond**. The two strands are held together how <strong style="font-size: 150%; color: red;">?</strong> hydrogen bonding.

A always pairs with T and G always pairs with C.  Therfore, total amount of (A+G)==total amount (T + C).

But GC content varies among species. GC content higher in bacteria -- especially in archae living in thermal vents. Why <strong style="font-size: 150%; color: red;">?</strong>  Because G-C 3 H-bonds so are stronger, takes more energy (temp) to separate them

Direction on DNA is important. One end is called 5' and the other end 3' -- these numbers come from the numbering of the carbon atoms of the doxyribose sugar. A molecule of DNA starst with phosphorylated 5' carbon and ends with hydroxyl group on a 3' carbon.

## Q4 

## Replication

Replication is semiconservative (meaning <strong style="font-size: 150%; color: red;">?</strong>) Each daughter DNA inherits 1 strand of original DNA + 1 new one (draw with colors).

The helix is opened and unwound by a **helicase** enzyme at the replication fork.  But if you've ever unwound strands of rope, you know this causes lots of twisting.  **Topoisomerase** releases tension by temporarily cutting, allowing spin, and resealing.

An enzyme called **DNA polymerase III** (there are other kinds of polymerase!) then adds nucloetides at the 3' end.  So DNA is syhtnesized 5'->3'.

Draw 

<img src="/Users/jri/Documents/courses/bis101/griffiths/ch07/figure_07_17.jpg" style="width: 500px;"/>

Anyone see problem with this <strong style="font-size: 150%; color: red;">?</strong>

**Leading strand** gets continuous replication 
**Lagging strand** cannot. Polymerase has to stop, come back and start again.

But polymerase can't start a new DNA, only extend it.  So different enzyme sets down a **primer** a short strand of DNA that primes DNA sythesis.

DNA pol I removes primer via its exonuclease activity (nculease = <strong style="font-size: 150%; color: red;">?</strong>), and then fills in gaps between **Okazaki fragments** and an enzyme called **DNA ligase** does <strong style="font-size: 150%; color: red;">?</strong> joins the bits together.

In total, can do this at ~1Kb/sec!

The whole process actually takes a number of proteins and protein complexes.  The **replisome** Figure 7-20 for bacteria, but very similar in Eukaryotes.  

* pol III
* beta clamp to keep pol III attached
* ligase for the okzazaki fragments 
* helicase and topoisomerase to unwind DNA
* single strand DNA could bind: single-stranded binding proteins prevent re-annealing

In E. coli, replication starts from one spot, the OriC or origin of replication. In Eukaryotes, DNA replicaiton starts from several origins at once, going in both directions.

<img src="/Users/jri/Documents/courses/bis101/griffiths/ch07/figure_07_24.jpg" style="width: 350px;"/>

What's another difference between Euk and bacteria <strong style="font-size: 150%; color: red;">?</strong> In Eukaryotes, DNA is in chromatin, so replisome has to be able to unwind DNA from nculeosomes to access it. Additional enzymes (e.g. chromatin assembly factor-1) required. 

* go back to ch01 and read. 

## The End

All this is well and good, but there's a problem. DNA gets shorter each time. Why <strong style="font-size: 150%; color: red;">?</strong>

Lagging strand requires addition of primers. Draw part of 

<img src="/Users/jri/Documents/courses/bis101/griffiths/ch07/figure_07_27.jpg" style="width: 350px;"/>

Because Pol 1 can only add to 3' end, it can't fix this problem at the **telomere**

**Telomerase**! Telomeres made up of repititve short fragments. Figure 7-28 in book explains details, but because it's repetitive telomerase can use a single known RNA fragment as a template to bind, create new overhang, elongate, repeat.  Then finally fill in with normal DNA pol in 5'->3' fashion.

Telomerase also binds to telomeres to form protective cap. Why needed <strong style="font-size: 150%; color: red;">?</strong> 

So cell doesn't think chromosome end is a DSB!

## Q5