#Transposable elements

##Genome Size

###C-value paradox

Genome size not correlate strongly with complexity (eukaryotes): C-value paradox
 
Varies 2000-fold in plants, from 64Mb in Genlisea to 120GB in Fritilleria (Humans are ~3Gb) Arabidopsis 120Mb, Pine trees 25Gb, Maize ~2.5Gb

No correlation in eukaryotes with number of genes (but pro vs. eu)

	So what is rest of genome? "Junk DNA" 
	Nematode ~5-10%, drosophila ~15%, humans ~50% repetitive/TE, maize  85%
    Some is simple repeats
        telmoere, centromere repeats       
    Most of it is TE

###Genome size & TEs in plants

Genome size and repeat % in plants. 

Majorit of plant DNA in world is TEs

##Transposable Elements

###Types
RNA elements Class I

- transcribe RNA, reverse transcribe into DNA
- copy and paste
- small to very large (20kb)
- **LTR long terminal repeat** (8% human genome)  
	- thought to derive from virus (gag/pol/env)  
- LINE: just a reverse transcriptase and RNA pol promoter
- **TSD** target site duplication b/c of sticky end cuts and replication to insert

DNA elements Class II

- cut and paste using transposase
- recognizes TIR
- usually small(ish)

Other weirdos (helitrons) 

- rolling circle replication
- resolution of heteroduplex -> sometimes integration, sometimes no 
- monstrous (50kb or larger)
- pikc up other genes, no target-site duplication/TIR

Autonomous & nonautonomous of each  

- autonomous: code for own proteins to transpose

		nonautonomous? can use proteins but not make
		
- nonautonomous can vary in size, incl. host genes
- SINE = short interspersed nuclear element

	- Alu SINEs in humans ~ 11% genome 
	
###Transposition mechanism

Class I

- LTR identical on insertions & used to date

Class II

	how replicate? jump ahead of rep. fork	
	DS break repair by homologous recombination mechanism 
	w/ TE on other copy	(or chromatid) as template
	
- draw in TIR in old spot in genome

Different elements like to go different places

##TE Impacts on the Genome

###Impacts 1

Insertions in genes, regulatory seq.

Retrotransposition

- new position & enhancer effects
- loss of introns, creation of **pseudogenes**
- insertion of new exons **exon shuffling**

###Impacts 2

New regulatory sequences (hopscotch tb1)

Rearrangement due to excision (recombination etc.) moves regulatory element

Exapted transposase (VDJ example)

Exon shuffling by reverse transcriptase

###Impacts 3

Recombination between TEs leads to genomic rearrangements
- gene loss
- translocations

Recombination within TEs mechanism for TE removal -- solo LTR

###Impacts 4

epigenetic silencing -> DICER & RISC

##Examples

###Kernels in maize

Nonautonomous insertion

Example of sectoring

- C purple; c no purple
- Ds = dissaciator; Ds+ = no TE; on same chromosome as C
- Ac activator elswhere in genome; Ac+ not present

###85% of a genome

islands of genes (500kb from GenoMaize)

Bronze locus region in 8 kinds of corn

### Epigenetic effects

transgene for red color in white fly

	why variegated when inserted into TE region of chromosome?  
	epigenetic silencing of heterochromatin
	
agouti color in mice

- weird outward transcript that reads into agouti gene
- level of color depends on epigenetic state of TE

morning glory

- spread of methlyation of nonautonomous MuLe can turn off color gene

maize w/ nonautonomous mu insertion nearby

- if autonomous mutator is active, transcribed

### V(D)J

Variable/Diverse/Joining regions of antibody genes in vertebrates

Responsible for immense diversity of antibodies B and C lymphocytes use to recognize foreign material and prime immune response

Recombinational Signal Sequence = TIR

Each exon is equivalent of Nonautonomous transposon

RAG proteins are coopted immobile versions of a TE w/ transposase

Different RSS interact w/ RAG in diff. cells for ~3Ã—10^11 possibilities

### Placenta!

MER20 TEs unique to non-placental mammals

Insertion of mer20 insulates chromsomes from spreading heterochromatin

Also containt binding sites for transcription factors imp. for regulating genes upregulated in placenta and during pregnancy

MER20 insertions thought to have rewired and patterned gene expression network in placental mammals

### Drosophila P

not found in wild in early 20th century, now in 100% of Drosophila in wild

causes hybrid sterility. 

- splice variant of P element epigentically shuts off P
- supressor protein not transmitted thru sperm
- so P-male by M-femal -> F1 w/ lots of transpositions & mutations -> lethal

P-element insertions provide mutational diversity for selection/evolution!

## Transposon tagging

can use known sequence of TE to identify unknown sequence of interest